thread/unix/thrdsup.c

```diff
@@ -66,12 +66,8 @@ intptr_t init_thread_library(void);
 static intptr_t zos_init_yielding(void);
 #endif
 intptr_t sem_getvalue_zos(j9sem_t s);
-intptr_t sem_init_zos(j9sem_t s, int pShared, int initValue);
 void call_omrthread_init(void);
-intptr_t sem_destroy_zos(j9sem_t s);
-intptr_t sem_wait_zos(j9sem_t s);
 intptr_t sem_trywait_zos(j9sem_t s);
```

Comment:
The declarations of `sem_getvalue_zos()` and `sem_trywait_zos()` should be in `thrdsup.h`.
The declaration of `call_omrthread_init()` can simply be removed - there are no references before the definition.